From: Shardul Deshpande <iamsharduld@gmail.com>
Subject: treewide: fix transposed "sign" typos and update spelling.txt
Date: Fri, 12 Jun 2026 23:46:32 +0530

Several comments transpose the letters in "assigned" and "unsigned",
spelling them with "sing" instead of "sign".  Correct all of them.

Of these, the misspelling of "assigned" is not yet flagged by checkpatch,
so also add it to scripts/spelling.txt.

The remaining matches of `grep -ri singed` are RISINGEDGE register and
enum names, not typos.

Link: https://lore.kernel.org/20260612181633.734458-1-iamsharduld@gmail.com
Signed-off-by: Shardul Deshpande <iamsharduld@gmail.com>
Suggested-by: Andrew Morton <akpm@linux-foundation.org>
Cc: Joe Perches <joe@perches.com>
Cc: SeongJae Park <sj@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 arch/arm64/boot/dts/qcom/sc7280-qcard.dtsi |    2 +-
 drivers/gpu/drm/drm_color_mgmt.c           |    2 +-
 drivers/scsi/elx/efct/efct_hw.h            |    2 +-
 drivers/scsi/isci/host.c                   |    2 +-
 drivers/scsi/isci/port.c                   |    2 +-
 drivers/scsi/isci/port_config.c            |    2 +-
 kernel/bpf/helpers.c                       |    2 +-
 scripts/spelling.txt                       |    1 +
 8 files changed, 8 insertions(+), 7 deletions(-)

--- a/arch/arm64/boot/dts/qcom/sc7280-qcard.dtsi~treewide-fix-transposed-sign-typos-and-update-spellingtxt
+++ a/arch/arm64/boot/dts/qcom/sc7280-qcard.dtsi
@@ -562,7 +562,7 @@ mos_bt_uart: &uart7 {
  *
  * This has entries that are defined by Qcard even if they go to the main
  * board. In cases where the pulls may be board dependent we defer those
- * settings to the board device tree. Drive strengths tend to be assinged here
+ * settings to the board device tree. Drive strengths tend to be assigned here
  * but could conceivably be overwridden by board device trees.
  */
 
--- a/drivers/gpu/drm/drm_color_mgmt.c~treewide-fix-transposed-sign-typos-and-update-spellingtxt
+++ a/drivers/gpu/drm/drm_color_mgmt.c
@@ -54,7 +54,7 @@
  *	&drm_crtc_state.degamma_lut.
  *
  * “DEGAMMA_LUT_SIZE”:
- *	Unsinged range property to give the size of the lookup table to be set
+ *	Unsigned range property to give the size of the lookup table to be set
  *	on the DEGAMMA_LUT property (the size depends on the underlying
  *	hardware). If drivers support multiple LUT sizes then they should
  *	publish the largest size, and sub-sample smaller sized LUTs (e.g. for
--- a/drivers/scsi/elx/efct/efct_hw.h~treewide-fix-transposed-sign-typos-and-update-spellingtxt
+++ a/drivers/scsi/elx/efct/efct_hw.h
@@ -59,7 +59,7 @@
 #define EFCT_HW_EQ_DEPTH		1024
 
 /*
- * A CQ will be assinged to each WQ
+ * A CQ will be assigned to each WQ
  * (CQ must have 2X entries of the WQ for abort
  * processing), plus a separate one for each RQ PAIR and one for MQ
  */
--- a/drivers/scsi/isci/host.c~treewide-fix-transposed-sign-typos-and-update-spellingtxt
+++ a/drivers/scsi/isci/host.c
@@ -2488,7 +2488,7 @@ struct isci_request *sci_request_by_tag(
  *    free remote node ids
  * @idev: This is the device object which is requesting the a remote node
  *    id
- * @node_id: This is the remote node id that is assinged to the device if one
+ * @node_id: This is the remote node id that is assigned to the device if one
  *    is available
  *
  * enum sci_status SCI_FAILURE_OUT_OF_RESOURCES if there are no available remote
--- a/drivers/scsi/isci/port.c~treewide-fix-transposed-sign-typos-and-update-spellingtxt
+++ a/drivers/scsi/isci/port.c
@@ -464,7 +464,7 @@ static enum sci_status sci_port_set_phy(
 {
 	/* Check to see if we can add this phy to a port
 	 * that means that the phy is not part of a port and that the port does
-	 * not already have a phy assinged to the phy index.
+	 * not already have a phy assigned to the phy index.
 	 */
 	if (!iport->phy_table[iphy->phy_index] &&
 	    !phy_get_non_dummy_port(iphy) &&
--- a/drivers/scsi/isci/port_config.c~treewide-fix-transposed-sign-typos-and-update-spellingtxt
+++ a/drivers/scsi/isci/port_config.c
@@ -259,7 +259,7 @@ sci_mpc_agent_validate_phy_configuration
 		if (!phy_mask)
 			continue;
 		/*
-		 * Make sure that one or more of the phys were not already assinged to
+		 * Make sure that one or more of the phys were not already assigned to
 		 * a different port. */
 		if ((phy_mask & ~assigned_phy_mask) == 0) {
 			return SCI_FAILURE_UNSUPPORTED_PORT_CONFIGURATION;
--- a/kernel/bpf/helpers.c~treewide-fix-transposed-sign-typos-and-update-spellingtxt
+++ a/kernel/bpf/helpers.c
@@ -3601,7 +3601,7 @@ __bpf_kfunc int bpf_copy_from_user_task_
 	return ret + 1;
 }
 
-/* Keep unsinged long in prototype so that kfunc is usable when emitted to
+/* Keep unsigned long in prototype so that kfunc is usable when emitted to
  * vmlinux.h in BPF programs directly, but note that while in BPF prog, the
  * unsigned long always points to 8-byte region on stack, the kernel may only
  * read and write the 4-bytes on 32-bit.
--- a/scripts/spelling.txt~treewide-fix-transposed-sign-typos-and-update-spellingtxt
+++ a/scripts/spelling.txt
@@ -176,6 +176,7 @@ assgined||assigned
 assiged||assigned
 assigment||assignment
 assigments||assignments
+assinged||assigned
 assistent||assistant
 assocaited||associated
 assocated||associated
_
